### ISSUE TITLE
Issues with requirejs

### ISSUE THREAD

**Issue by kuzmycz** [chartist-js/chartist#872]
If I use version 0.1.0 I have problems with chartist. It seems that the module does not resolve in require.js (Get chartist undefined when trying to access Chartist.plugins). If I drop the version back to 0.9.7 everything works fine. (Note I am also using chartist-plugin-fill-donut)

**Comment by arabsight** [comment]
adding 'Chartist' to define() broke my aurelia code which uses requirejs.

**Comment by aixnip** [comment]
I also have the same issue. Had to use v0.9.7 thanks to your info...

**Comment by powerbuoy** [comment]
0.9.7 works for me too.

**Comment by gionkunz** [comment]
Probably related to #894

**Comment by andrewdbate** [comment]
I can confirm that I am facing the same problem.

I just spent a few hours trying to work out what was going on. It would be helpful if there was some info about using chartist with react.js on the web page (similar to the chart.js docs have [here](http://www.chartjs.org/docs/latest/getting-started/integration.html), explinaing that v0.9.7 must be used).

**Comment by cedeber** [comment]
It still doesn't work with requirejs.

To solve the problem, instead of  https://github.com/gionkunz/chartist-js/blob/5cbd88a2a551134e81e12414ca18d5bf57982ba2/dist/chartist.js#L4

write (as on v0.9.7)
```js
define([], function () {
```

**Comment by pimbrouwers** [comment]
Even easier, just use it as it's named.

Like: 
`require('Charist');`

or

`define(['Chartist'], function(Chartist){});`

**Comment by rosgzc** [comment]
Having the same issue with 0.11.0, so had to revert to using 0.9.7, which works fine.

**Comment by arabsight** [comment]
are we supposed to be stuck on that old version forever.

**Comment by andrewdbate** [comment]
Looking at the open issues, it appears that Chartist is no longer actively maintained.

Does anyone have any recommendations for alternatives?

**Comment by pimbrouwers** [comment]
@rossgzc @arabsight @andrewdbate did you guys try the solution I provided above? Most often with require it's misconfiguration in the shim that wreaks havoc on the ability for it to work correctly (the main reason I switched to browserify months ago). 

I found the best way for require to work, was to have an npm script that moves all the JavaScript dependencies into your js "root", with require.js sitting next to all the external dependencies it relies on it almost eliminates the need to configure anything. This setup is how I had most luck with require on the dozen enterprise projects I used it for.
